docs/code_search/reference/structural.mdx

```diff
@@ -1,5 +1,7 @@
 # Structural search
 
+<Callout type="note"> Changed in version 5.3. Structural search is disabled by default. To enable it, ask your site administrator to set `experimentalFeatures.structuralSearch = "enabled" in site configuration.</Callout>
```

Comment:
There are a lot of references to structural search in the docs. I put a single note at the top of this central doc to keep the noise on other pages to a minimum. Let me know if I you think this is sufficient.